libs/types/src/smartcard_auth/dipped_smartcard_auth.ts

```diff
@@ -30,17 +30,19 @@ export interface RemoveCard {
   readonly user: SystemAdministratorUser | ElectionManagerUser;
 }
 
+export type CardSummaryNotReadyStatus = 'no_card' | 'error';
```

Comment:
Great question about the organization of types across `libs/utils` and `libs/types` - I'm tempted to say that we should move the card types from `libs/utils/src/types.ts` to a `libs/types/src/card.ts` file

Trying to import the types from `libs/utils` into `libs/types` will result in a circular dependency, since `libs/utils` also imports `libs/types`. I think of `libs/types` as one of the most fundamental, low-level packages in VxSuite. Other VxSuite packages should depend on it, but it should depend on no other VxSuite packages

If you'd like to handle this in a separate PR, totally fine by me!